Modules/IxNetRestApiPacketCapture.py

```diff
@@ -244,21 +242,12 @@ def packetCaptureGetCurrentPacketsHex(self, getUpToPacketNumber=20):
                 self.ixnObj.logInfo('Getting captured packet index number: {}/{}'.format(packetIndex, getUpToPacketNumber))
 
                 if totalDataCapturedPackets > 0:
-                    data = {'arg1': vport+'/capture/currentPacket', 'arg2': packetIndex}
-                    response = self.ixnObj.post(self.ixnObj.sessionUrl+'/vport/capture/currentPacket/operations/getpacketfromdatacapture',
-                                                data=data, silentMode=False)
-                    self.ixnObj.waitForComplete(response, self.ixnObj.sessionUrl+'/vport/capture/currentPacket/operations/getpacketfromdatacapture/'+response.json()['id'])
+                    vport.Capture.CurrentPacket.GetPacketFromDataCapture(Arg2 = packetIndex )
 
                 if totalControlCapturedPackets > 0:
-                    data = {'arg1': vport+'/capture/currentPacket', 'arg2': packetIndex}
-                    response = self.ixnObj.post(self.ixnObj.sessionUrl+'/vport/capture/currentPacket/operations/getpacketfromcontrolcapture',
-                                                data=data, silentMode=False)
-
-                    self.ixnObj.waitForComplete(response, self.ixnObj.sessionUrl+'/vport/capture/currentPacket/operations/getpacketfromcontrolcapture/'+response.json()['id'])
-                
-                data = {'arg1': '-packetHex'}
-                response = self.ixnObj.get(self.ixnObj.httpHeader+vport+'/capture/currentPacket', data=data, silentMode=False)
-                packetHex = response.json()['packetHex']
+                    vport.Capture.CurrentPacket.GetPacketFromControlCapture(Arg2=packetIndex)
+
+                packetHex = vport.Capture.CurrentPacket.PacketHex
                 capturedData[eachTypeOfCaptures][packetIndex]=packetHex
```

Comment:
`=` operator for assignment should have space and no space in argument. Follow PEP standard